src/Raven.Client/Documents/Session/IAsyncSessionDocumentTimeSeries.cs

```diff
@@ -15,66 +15,83 @@ namespace Raven.Client.Documents.Session
     /// <summary>
     ///     Advanced async TimeSeries session operations
     /// </summary>
-    public interface IAsyncSessionDocumentTimeSeries : 
-        ISessionDocumentAppendTimeSeriesBase, 
+    public interface IAsyncSessionDocumentTimeSeries :
+        ISessionDocumentAppendTimeSeriesBase,
         ISessionDocumentDeleteTimeSeriesBase,
         IAsyncTimeSeriesStreamingBase<TimeSeriesEntry>
     {
+        /// <summary>
+        /// Retrieves a range of entries from a single time series.<br/>
+        /// <seealso href="https://ravendb.net/docs/article-page/6.0/csharp/document-extensions/timeseries/client-api/session/get/get-entries#timeseriesfor.get"/>
```

Comment:
Please make sure that _all_ links to the documentation are referenced via `DocumentationUrls.cs`

Reply:
done